### ISSUE TITLE
ABO terminal ('leaf') terms

### ISSUE THREAD

**Issue by pmidford** [obo-behavior/behavior-ontology#49]
This is a list of terms from ABO that we (Anne, Sue, Peter) wish to include in NBO.  These 
terms are all subsumed by (terminal on the tree relative to) existing NBO terms.  There 
are a small number of terms that we propose to insert within the NBO hierarchy, but 
those will come in a later issue.

Each of these terms has been assigned an identifier from the block of identifiers assigned
to @dosumis during the workshop in DC in October 2015.  We have used only 300 of
this range, so there will be plenty for VFB to use in the future.  These include approximately
10 terms that were not in the version of ABO that we used for the preliminary merge at that
workshop.  I have manually assigned these the next IDs in sequence.

## Terms to add

| Term                              | Parent Term                | Term ID     |
|  -------------------------------- | -------------------------- | ----------- |
| Agonistic alarm                    | Agonistic behavior          | NBO_0020145 |
| Agonistic buffering                | Agonistic behavior          | NBO_0020112 |
| Agonsitic chase                    | Agonistic behavior          | NBO_0020141 |
| appeasement                       | Agonistic behavior          | NBO_0020279 |
| assessment behavior               | Agonistic behavior          | NBO_0020093 |
| coercion                          | Agonistic behavior          | NBO_0020063 |
| contact aggression                | Agonistic behavior          | NBO_0020033 |
| displacement behavior             | Agonistic behavior          | NBO_0020255 |
| escalation                        | Agonistic behavior          | NBO_0020182 |
| exclusion                         | Agonistic behavior          | NBO_0020220 |
| injure (injuring)                 | Agnostic behavior          | NBO_0020111 |
| kill (killing)                    | Agonistic behavior          | NBO_0020018 |
| threat                            | Agonistic behavior          | NBO_0020108 |
| triumph                           | Agonistic behavior          | NBO_0020239 |
| Aposematism                       | Protective behavior        | NBO_0020287 |
| bluff                             | Protective behavior        | NBO_0020031 |
| create refuge                     | Protective behavior        | NBO_0020105 |
| misdirect predator                | Protective behavior        | NBO_0020219 |
| deploy defense                    | Protective behavior        | NBO_0020117 |
| (predator) detection notification | Protective behavior        | NBO_0020289 |
| display condition (to predator)   | Protective behavior        | NBO_0020077 |
| distress signaling                | Protective behavior        | NBO_0020149 |
| enhance crypsis                   | Protective behavior        | NBO_0020139 |
| feign death                       | Protective behavior        | NBO_0020199 |
| flee                              | Protective behavior        | NBO_0020268 |
| inspect predator                  | Protective behavior        | NBO_0020223 |
| mimicry enhancement               | Protective behavior        | NBO_0020245 |
| jamming                           | Protective behavior        | NBO_0020079 |
| occupy refuge                     | Protective behavior        | NBO_0020059 |
| promote confusion                 | Protective behavior        | NBO_0020251 |
| reduce activity                   | Protective behavior        | NBO_0020175 |
| attack predator                   | Protective behavior        | NBO_0020230 |
| vigilance                         | Protective behavior        | NBO_0020037 |
| warn other prey                   | Protective behavior        | NBO_0020015 |
| Nutrient acquisition              | Hunger motivation behavior | NBO_0020282 |
| Nutrient locating                 | Hunger motivation behavior | NBO_0020188 |
| Nutrient preparation              | Hunger motivation behavior | NBO_0020291 |
| Resource advertisement            | Hunger motivation behavior | NBO_0020088 |
| Social acquisition of food        | Hunger motivation behavior | NBO_0020177 |
| capturing (behavior)              | Predatory behavior         | NBO_0020292 |
| chase prey                        | capturing behavior         | NBO_0020040 |
| capturing prey                    | capturing behavior         | NBO_0020293 |
| construct capturing device        | capturing behavior         | NBO_0020137 |
| flush prey                        | capturing behavior         | NBO_0020045 |
| stalk prey                        | capturing behavior         | NBO_0020160 |
| aiding digestion                  | Nutrient acquisition       | NBO_0020007 |
| dealing with toxins               | Nutrient acquisition       | NBO_0020142 |
| food storage                      | Nutrient acquisition       | NBO_0020068 |
| caching behavior                  | Food storage               | NBO_0020294 |
| trail following to resource       | Nutrient locating          | NBO_0020096 |
| food exposure                     | Nutrient locating          | NBO_0020170 |
| recovery from cache               | Nutrient locating          | NBO_0020144 |
| food extraction                   | Nutrient preparation       | NBO_0020286 |
| food processing                   | Nutrient preparation       | NBO_0020295 |
| suitability assessment            | Nutrient preparation       | NBO_0020091 |
| beg food                          | Social acquisition of food | NBO_0020034 |
| scrounge                          | Social acquisition of food | NBO_0020296 |
| scavenge                          | Social acquisition of food | NBO_0020297 |
| share food                        | Social acquisition of food | NBO_0020173 |
| cooling behavior                  | Thermoregulation behavior  | NBO_0020159 |
| mate finding behavior             | Reproductive behavior      | NBO_0020084 |
| sexual interference               | Reproductive behavior      | NBO_0020298 |
| mate rejection                    | Reproductive behavior      | NBO_0020299 |
| sexual harassment                 | Reproductive behavior      | NBO_0020254 |
| hunt for mate                     | mate finding behavior      | NBO_0020272 |
| reproductive status assessment    | Mate finding behavior      | NBO_0020010 |
| mate sampling                     | Mate finding behavior      | NBO_0020281 |
| insert genital plug               | sexual interference        | NBO_0020047 |
| sperm scraping                    | sexual interference        | NBO_0020109 |
| breeding site selection           | Mating behavior            | NBO_0020273 |
| courtship begging                 | Courtship behavior         | NBO_0020098 |
| courtship feeding                 | Courtship behavior         | NBO_0020264 |
| mating appeasement                | Courtship behavior         | NBO_0020253 |
| sexual cannibalism                | Courtship behavior         | NBO_0020101 |
| mating solicitation               | Mating receptivity         | NBO_0020241 |
| broadcast advertisement           | Sexual display behavior    | NBO_0020023 |
| build display structure           | Sexual display behavior    | NBO_0020242 |
| mate guarding                     | Post mating behavior       | NBO_0020054 |
| pair affirmation                  | Post mating behavior       | NBO_0020071 |
| mouth brooding                    | Post mating behavior       | NBO_0020102 |
| gamete release function           | Sexual Activity            | NBO_0020066 |
| facilitating oviposition          | Sexual Activity            | NBO_0020048 |
| insemination                      | copulation                 | NBO_0020121 |
| mating amplexus                   | gamete release function    | NBO_0020204 |
| spawning                          | gemete release function    | NBO_0020174 |
| spermatophore deposition          | gamete release function    | NBO_0020266 |
| agnostic alarm                    | behavioral alarm           | NBO_0020145 |

## Definitions

Agnostic alarm - Alarm behavior triggered by agonistic actions or postures

Agnostic buffering - Carrying or manipulation of infants or equivalent items to reduce chances of aggression on carrier by other animals

Agnostic chase - Chasing of one animal by another during an agonistic conflict

appeasement - Behaviors that appear to signifiy lack of hostility and to reduce chances of attack

assessment behavior - A behavior on the part of one individual associated with evaluating the fighting ability or correlates of fighting ability of a potential opponent

capturing prey - Behavior that results in ending flight of prey with successful predation

coercion - Forcible alteration of behavior of one animal by another.

contact aggression - Agonistic behavior involving direct physical contact including (but not limited to) biting, kicking, grabbing.

displacement behavior - An acitvity that seems irrelevant to its behavioral context to a human observer and often occurs when its performer is in a conflict situation

escalation - Increasing intensity of directed agonistic behavior.

exclusion - Behavior that serves to exlude an animal from a group

injure (injuring) - Behavior in which one animal causes bodily damage to another.

kill (killing) - Behavior in which one animal causes the death of another.

threat - Behavior that signifies hostility and predicts an increased probability of attack

triumph - Display behavior by the winner of a prior agonistic contest.

Aposematism - Conspicuous postures or signals (often coloration) that potential predators learn to associate with strong defense mechanisms of performer

bluff - Threat that will not be acted upon.

create refuge - Construction of a structure in which builder can benefit from reduced predation risk and/or environmental stress.

misdirect predator - Behaviors directed towards presumed predators to redirect their attention away from a location of importance to the performer

deploy defense - use of behavioral tactics or chemical weapons to deter predation.  Example: flashing eye spots (behavioral tactic) or expulsion of noxious liquid (chemical)

(predator) detection notification - Postures or signals that can communicate to predator that it has been detected by performer

display condition (to predator) - Performance of behaviors that demonstrate physical condition of performer to a detected predator

distress signaling - Signals emitted by an animal trapped or captured by a presumed predator

enhance crypsis - Behaviors that reduce detectability of performer against background

feign death - Become immobile and reduce other evidence of living state

flee - Depart [from] current location of predator

inspect predator - Conspicuous orientation towards and/or approach to detected predator

mimicry enhancement - Behaviors, anatomical structures, or signals that imitate aposematic animals, plants, or inanimate objects that are undesirable to predators of performer

jamming - Emission of signals in such a way as to prevent effective use of signals emitted by other parties

occupy refuge - Currently reside in constructed or existing structure that provides reduced predator and/or environmental risk

promote confusion - Erratic movements or similar behaviors that would confuse a predator.

reduce activity - Halting or diminution of activity to minimize detection by predator.

attack predator - Perform physically violent acts on a presumed predator

vigilance - Surveillance for presence or approach of predators

warn other prey - Postures or signals that alert other animals to presence or approach of a predator.

Nutrient acquisition - Behaviors related to acquisition and utilization of external resources.

Nutrient locating - Movement directed to the detection of new resources.

Nutrient preparation - Behavior that precedes and facilitates attaining food

Resource advertisement - Postures or signals performed to other animals to indicate the presence, and in some cases, the location, quantity, and quality, of a resource

Social acquisition of food - Obtaining food that has been procured by other conspecifics or heterospecifics.

capturing (behavior) - Suite of behaviors involved in securing a prey item once it has been located.

chase prey - Active pursuit of fleeing prey.

capturing prey - Behaviors that result in ending flight of prey with successful predation.

construct capturing device - Building of a trap, web, net, or other structure to help ensnare potential food items

flush prey - Behaviors in which predator forces potential prey to emerge from cover.

stalk prey - Approach to prey in manner designed to avoid early detection by prey.

aiding digestion - consumption of specific compounds that facilitate processing of food items

dealing with toxins - The identification, avoidance, or neutralization of toxic substances in food

food storage - Caching or hoarding of food for later use and for protection from competitors, or use of specific substrates for cultivation.

caching behavior - The act of putting a food resource in a location in which it can be stored or protected until a future time

follow trail to resource - Locomotion to a resource guided by visual or olfactory cues created by a prior finder and/or user of that resource.

food exposure - Behaviors that reveal a hidden food resource, such as digging, cracking, or breaking.

recovery from cache - Removal of food resource from previously-stored supply of food.

food extraction - Removal of food items from location by use of appendage or tool.

food processing - Behaviors that prepare food for consumption, such as nut-cracking or peeling

suitability assessment - Sensory assessment of a resource, (usually using olfaction or taste), for suitability of subsequent usage

beg food - Perform behaviors that may induce one or more other animals to share food with the performer

scrounge - Uninvited taking of food from another animal

scavenge - Behavior associated with locating and acquiring food that has been killed or procured by another animal.

share food - Behaviors in which animals solicit or allow other animals to utilize a resource they currently control

cooling behavior - Behaviors that serve to reduce body temperature as by moving to shade or panting

mate finding behavior - Active search for possible mates

sexual interference - Usually interference by members of one sex in the completion of courtship or mating activities by one or more members of the same sex

mate rejection - Behavior associated with refusing courtship or solicitation behaviors by avoidance, retreat, or other active repulsion of suitor. 

sexual harassment - Any of several behaviors directed towards one animal unwilling to mate by another eager to do so

hunt for mate - active searching of potential mate

reproductive status assessment - Assessment of cues and signals from encountered individual to determine reproductive condition

mate sampling - General category of behaviors including sampling of medium to detect presence of possible mates, and sampling of encountered individuals to determine mate suitability

insert genital plug - Behavior associated with deposition of material by male following ejaculation to block access to female's reproductive tract during her future matings.

sperm scraping - use of specialized male organs to remove sperm already deposited in female's reproductive tract.

breeding site selection - Actions resulting in establishment or use of one locale for reproductive activity

courtship begging - Behavior by one animals soliciting food provision by a potential mate

courtship feeding - provision of food to a potential mate

mating appeasement - Behavior that serves to demonstrate non-aggressive intent by one member of a courting or potentially mating pair to the other.

sexual cannibalism - Consumption of one member of a mating pair, usually the male, usually after the mating

mating solicitation - Performance of a behavior that indicates the performer's willingness to mate or bond

broadcast advertisement - Signal or behavior that reveals presence of the signaller directed to multiple receivers and not limited to particular receivers

build display structure - Creation of  or modification of a structure or area that is used in attraction of mates

mate guarding - One member of a recently mating pair prevents the other from subsequent matings with other individuals

pair affirmation - displays or other behaviors performed by pairs in longer term pair bonds to advertise relationship to each other and to other animals nearby

mouth brooding - Process of retaining developing eggs in the mouth cavity

gamete release function - Facilitating release or transfer of gametes

facilitating oviposition - Deposition of eggs either into the medium, or onto a substrate

insemination - Insertion of sperm not in a spermatophore into female body

mating amplexus - Spawning in which male and female are clasped or attached to ensure close proximity of released gametes

spawning - Release of gametes one or both sexes into the ambient medium

spermatophore deposition - Deposition of spermatophore either on substrate or in body of female

agnostic alarm - Alarm behavior triggered by agonistic actions or postures

Mentioning @swmargulis here so she sees this issue.

**Comment by dosumis** [comment]
Agnostic or agonistic?

**Comment by aclark-binghamton-edu** [comment]
yikes  Agonistic!



Anne B. Clark, Ph.D.
Biological Sciences
Binghamton University
Binghamton, NY 13902
1-607-777-6228, Fax -777-6521
C. 607-222-0905

We are speaking where we stand and we shall stand afterwards in the presence of what we have said.--Wendell Berry—"Standing By Words"

**Comment by pmidford** [comment]
Ugh. Fixed.

**Comment by dosumis** [comment]
CC @matentzn - We should work on patternising these following https://docs.google.com/document/d/1eSCjQfR5BujtUYw4W2pYOL6rhPPMycJd8u95X1-5U7Q/edit

**Comment by pmidford** [comment]
These terms have been added, so I will close this.  However, there are at least two issues related to these terms: one is David's comment about patternizing them, a second is that I've noticed some incorrect assertions of disjointness among these terms that need to be discussed.  I'll write a ticket about the second and unless David wants to create the ticket for patternizing (which I generally agree with), I'll write that later this week.
